### PR TITLE
Add whalefin

### LINKED ISSUE #6498: Add Whalefin
### Site name

Whalefin

### Site URL

https://www.whalefin.com/

### Documentation URL

https://support.whalefin.com/hc/en-us/articles/4403359735961-How-to-enable-Google-Authenticator-2FA-on-WhaleFin-App-

### Supported 2FA methods

- [X] TOTP (RFC-6238, Google Authenticator)
- [ ] Proprietary Software (Authy, Duo, etc.)
- [ ] FIDO2/U2F/WebAuthn security keys
- [ ] Proprietary hardware (Yubico OTP, RSA keys, etc.)
- [ ] Phone calls
- [X] SMS tokens
- [X] Email tokens

### Eligibility

- [X] I have checked that the site meets 2fa.directory's [contributing guidelines and site criteria](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md)
- [ ] The site is ranked within the top 200,000 sites on [Alexa](https://www.alexa.com/siteinfo/) or has a substantial social media following.
- [X] The site does not contain or promote content that violates 2fa.directory's [excluded categories and websites guidelines](https://github.com/2factorauth/twofactorauth/blob/master/EXCLUSION.md), including pornographic, discriminatory, or unlawful content.
- [X] The issue I'm creating is not a duplicate of an existing [issue](https://github.com/2factorauth/twofactorauth/issues).
- [X] The issue I'm creating is not a duplicate of an existing [pull request](https://github.com/2factorauth/twofactorauth/pulls)

### Additional information

The Alexa rank is now  24,120, probably because of the rebranding, still think it's worthy to be added in 2fa.directory, also recomanded by our clients.

### PR DESCRIPTION
**Pull request by CCharlieLi**

### CODE REVIEW

File: entries/w/whalefin.com.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "Whalefin": {
+    "domain": "whalefin.com",
+    "additional-domains": ["ambergroup.io"],
```
Comment (phallobst) [inline entries/w/whalefin.com.json]: This is just the parent company - we don't really need it here.
```suggestion
```

File: entries/w/whalefin.com.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "url": "https://www.whalefin.com",
```
Comment (phallobst) [inline entries/w/whalefin.com.json]:
```suggestion
```

File: entries/w/whalefin.com.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    "tfa": [
+      "sms",
+      "email",
+      "totp"
+    ],
+    "documentation": "https://support.whalefin.com/hc/en-us/articles/4403359735961-How-to-enable-Google-Authenticator-2FA-on-WhaleFin-App-",
+    "recovery": "https://support.whalefin.com/hc/en-us/articles/4403359754009-How-to-recover-Google-Authenticator-enabled-on-WhaleFin-App-",
```
Comment (phallobst) [inline entries/w/whalefin.com.json]:
```suggestion
    "documentation": "https://support.whalefin.com/hc/en-us/articles/4403359735961",
    "recovery": "https://support.whalefin.com/hc/en-us/articles/4403359754009",
```

File: img/w/whalefin.com.svg [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+<defs>
+<linearGradient id="paint0_linear_717_18435" x1="46.6192" y1="0.721958" x2="9.63992" y2="0.721958" gradientUnits="userSpaceOnUse">
+<stop stop-color="#0575E6"/>
+<stop offset="1" stop-color="#02F260"/>
+</linearGradient>
+</defs>
+</svg>
```
Comment (phallobst) [inline img/w/whalefin.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="38.1" height="46.43" fill="none"><path fill="url(#a)" d="M11.45 10.13c-2.85 1.69-6.1 4.12-8.13 6.9-1.55-3.96-.8-8.68 1.26-12.77A15.4 15.4 0 0 1 7.76 0a8.84 8.84 0 0 0 3.7 10.13zm6.38 36.3-.9-.05c-.2-.02-.4-.07-.6-.07A24.8 24.8 0 0 1 33.68 33.9c1.05-.17 2.1-.27 3.17-.3-.6 1.6-1.4 3.1-2.38 4.5a19.72 19.72 0 0 0-13.1 8.1 19.7 19.7 0 0 1-3.54.22zm4.5-.45a19.07 19.07 0 0 1 11.57-7.16 19.66 19.66 0 0 1-11.56 7.16zm14.84-13.34A25.59 25.59 0 0 0 15.44 46.2C7.6 44.96 1.9 38.87.35 31.61A19.12 19.12 0 0 1 3.4 17.03c1.2 7.76 9.29 9.6 17.04 9.6h17.62c.04.2.04.43.04.65a19.91 19.91 0 0 1-.93 5.36z"/><defs><linearGradient id="a" x1="46.62" x2="9.64" y1=".72" y2=".72" gradientTransform="translate(-8.95)" gradientUnits="userSpaceOnUse"><stop stop-color="#0575E6"/><stop offset="1" stop-color="#02F260"/></linearGradient></defs></svg>
```

File: entries/w/whalefin.com.json [hunk 5]
```diff
@@ -0,0 +1,17 @@
+{
+  "Whalefin": {
+    "domain": "whalefin.com",
+    "additional-domains": ["ambergroup.io"],
+    "url": "https://www.whalefin.com",
+    "tfa": [
+      "sms",
+      "email",
+      "totp"
```
Comment (phallobst) [inline entries/w/whalefin.com.json]: The documentation only mentions Google Authenticator, i.e. `totp`. Do you have further information about 2FA via `email` and `sms`?

**Review by phallobst — CHANGES_REQUESTED** [review]

File: entries/w/whalefin.com.json [hunk 6]
```diff
@@ -0,0 +1,17 @@
+{
+  "Whalefin": {
```
Comment (phallobst) [inline entries/w/whalefin.com.json]:
```suggestion
  "WhaleFin": {
```

**Review by phallobst — CHANGES_REQUESTED** [review]

File: entries/w/whalefin.com.json (hunk 5 above)
Comment (CCharlieLi) [reply entries/w/whalefin.com.json]: thanks for reviewing, and yes - there are a few pages on sms/email 2FA for sensitive actions
https://support.whalefin.com/hc/en-us/articles/4403359729945
https://support.whalefin.com/hc/en-us/articles/4409939331737
https://support.whalefin.com/hc/en-us/articles/4409952407321

Comment (phallobst) [reply entries/w/whalefin.com.json]: These "sensitive actions" do not seem to include logging in, so it's _authorization_ rather than _authentication_.

Comment (CCharlieLi) [reply entries/w/whalefin.com.json]: 2FA for login can also be regarded as authentication tho, depending on how you see and use it in different scenarios, the 2FA through sms/email is only required when taking actions that may cause losing money.

Comment (phallobst) [reply entries/w/whalefin.com.json]: That's what I mean - `sms` and `email` are only used for _authorization_; `totp` can stay.

Comment (CCharlieLi) [reply entries/w/whalefin.com.json]: done

**Review by phallobst — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**